services/workflows-service/src/data-analytics/data-analytics.repository/evaluate-transactions-against-dynamic-rules.ts

```diff
@@ -1,42 +1,29 @@
 import { PrismaClient, Prisma, TransactionDirection, PaymentMethod } from '@prisma/client';
 import { Sql } from '@prisma/client/runtime';
+import { AggregateType } from '../consts';
+import type { InlineRule, TransactionsAgainstDynamicRulesType } from '../evaluate-types';
 
 const prisma = new PrismaClient();
 
-enum AggregateType {
-  SUM = 'SUM',
-  AVG = 'AVG',
-  COUNT = 'COUNT',
-  MAX = 'MAX',
-  MIN = 'MIN',
-}
+const ruleToEvaluateFunction = {
+  [evaluateTransactionsAgainstDynamicRules.name]: (options: InlineRule) => {
+    evaluateTransactionsAgainstDynamicRules(options);
+  },
+};
 
-export const evaluateTransactionsAgainstDynamicRules = async ({
+export async function evaluateTransactionsAgainstDynamicRules({
```

Comment:
We have a lint rule against this, why'd you change it?

Reply:
will change it in my next pr